src/server/pfs/cmds/cmds.go

```diff
@@ -328,7 +328,7 @@ Files can be read from finished commits with get-file.`,
 			if len(args) == 3 {
 				path = args[2]
 			}
-			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard())
+			fileInfos, err := pfsclient.ListFile(apiClient, args[0], args[1], path, "", shard(), true)
```

Comment:
I see, that behavior for pachctl seems reasonable. The name `recurse` is a little weird imo though. Since I'd expect it to mean that we actually return records for the files in subdirectories not just include them in size counts. Do you think there's another name that describes this better? I can't think of one of the top of my head.